Sources/Spezi/Spezi.docc/Spezi.md

```diff
@@ -87,6 +91,9 @@ You can learn more about modules in the <doc:Module> documentation.
 
 ## Topics
 
+### Ecosystem Overview
```

Comment:
We shoulda also aim to incorporate some general documentation on how to get started with Spezi & your learnings of embedding yourself in the ecosystem as part of this PR to help people who are new to the ecosystem to help them answer some clarifying questions.